src/blacklist.txt

```diff
@@ -66,4 +66,145 @@ ws://mine.nahnoji.cz/*
 *://miner.pr0gramm.com/pm.min.js*
 *://miner.cryptobara.com/client/*
 *://digger.cryptobara.com/client/*
-ws://digger.cryptobara.com/*
\ No newline at end of file
+ws://digger.cryptobara.com/*
+23.239.31.174
```

Comment:
Not sure if that syntax is supported by the webRequest API. Also the goal of No Coin is to target specific resources rather than entire domains or IPs, except if proven that the only purpose of that domain/ip is to host a web mining script. Do you maybe have a list of domains associated to these IPs?